backend/Dockerfile

```
@@ -1,11 +1,28 @@
 FROM ubuntu:18.04
 
 RUN apt-get update && apt-get install -y python3 python3-pip libpq-dev gettext python-dev
-WORKDIR /match4everyone-backend
-COPY requirements.txt /match4everyone-backend/requirements.txt
+
+WORKDIR /backend
+COPY requirements.txt /backend/requirements.txt
 RUN pip3 install -r requirements.txt
-COPY requirements.prod.txt /match4everyone-backend/requirements.prod.txt
+COPY requirements.prod.txt /backend/requirements.prod.txt
 RUN pip3 install -r requirements.prod.txt
 
+COPY . .
+
+# Set Django Settings to production, for the built docker image this will use whitenoise to collect the static files
+# Our app will refuse to run without a SECRET_KEY. collectstatic parses the configuration module and fails
+# if no key is set. To prevent this a dummy key is used. In production mode this needs to be supplied
+# using an env file in the docker-compose.yml
+ENV DJANGO_SETTINGS_MODULE="match4everyone.settings.production" SECRET_KEY="CzXG4ItUiwLUfTH2abQQ0qTzMSRiiDni"
+RUN python3 manage.py makemessages --no-location --ignore 00_old_m4h_matching_code
```

In our readme we tell the user to use this command:
`[...] makemessages -l de --no-location`

Shouldn't we use the same in the docker file itself?

```suggestion
RUN python3 manage.py makemessages --no-location
```